guide/general/faq.md

```diff
@@ -44,13 +44,25 @@ Since the cooldown of {{ site.data.spell.meteor }} is the same as the duration o
 
 Not that we're aware of. The Elementals' autocast settings are shared between the Elemental pets. If you deactivate or activate it on one button, the matching button of the other Elementals will share that setting. We hope, that this will be changed.
 
-~~It was changed! 8.3 introduces separate toggle states for all three Elementals and their casts. This now should no longer pose an issue.~~ The fix was reverted just before 8.3 went live.
+~~It was changed! 8.3 introduces separate toggle states for all three Elementals and their casts. This now should no longer pose an issue.~~ The fix was reverted just before 8.3 went live due to bugs introduced from the implementation.
 
 ### Q: "Do I use Lava Burst during Storm Elemental?"
 
 Yes. No. It depends. Check out the [Priority List]({{ site.baseurl }}{% link guide/general/priority_list.md %}) and special notes on that website for in depth information.
 
 ### Q: "What are some useful Macros?"
+ 
+### Q: "What is funnelling?"
+ 
+Funneling is the act of casting {{ site.data.spell.fs }} on multiple targets in order to generate {{ site.data.spell.ls }} procs with the intention of increasing your damage on one target.
+ 
+### Q: "How does {{ site.data.talent.ecs }} interact with proc to abilities?"
+ 
+ {{ site.data.talent.ecs }} does not interact with procs to abilities; {{ site.data.talent.ecs }} will only echo an ability at its base.
```

Comment:
I dont think this is understandable tbh. I also dont have too much of an idea of how to phrase it better